torchci/components/benchmark/compilers/ModelPanel.tsx

```diff
@@ -138,6 +138,10 @@ export function ModelPanel({
     };
   });
 
+  console.log(lData);
```

Comment:
I guess you want to clear these lines before commiting

Reply:
Right, it is still WIP while I am debugging the queries